recipe/meta.yaml

```diff
@@ -7,32 +8,52 @@ package:
 source:
   url: https://github.com/google/snappy/archive/{{ version }}.tar.gz
   fn: snappy-{{ version }}.tar.gz
-  sha256: 6fa92cde5b2caefd0d9a60336991ba42e5a7ddc3bdc36c5610451373751d0495
+  sha256: {{ sha256 }}
   patches:
     # 1.1.6 crashes because it tries to install README, but the file is now
     # named README.md. This patch is submitted upstream:
     # https://github.com/google/snappy/pull/46
     - fix-autoconf-readme.patch
+    # Have snappy-unittest not run the slow-ish microbenchmarks, but just
+    # the quick correctness tests, without bothering with gflags (which didn't
+    # work out of the box). This is the recommended method in the README.
+    - skip-microbenchmarks.patch
+    # Patch CMakeLists to build a static library as well. Not currently
+    # available upstream, but a similar change is included in:
+    # https://github.com/google/snappy/pull/38
+    - build-static-lib.patch
```

Comment:
The patch here is not really the same as PR ( https://github.com/google/snappy/pull/38 ). Given the patch added here is much cleaner and easier to reason about, I think it has a better chance of being accepted. We should PR it anyways to see if they will accept it.